lib/nexpose/connection.rb

```diff
@@ -44,20 +44,29 @@ class Connection
     # The last XML response received by this object, useful for debugging.
     attr_reader :response_xml
 
+    # Enforcement of certificate validation to a CA
+    attr_reader :allow_untrusted
```

Comment:
Not yet, I would like to extend this to be usable but it starts to make this more of a breaking change, can remove or expand depending on how discussion goes.